### ISSUE TITLE
[Bluesky] Default Permissions Not Recognized 

### ISSUE THREAD

**Issue by AndyHee** [friendica/friendica#13521]
- [X ] I have searched open and closed issues for duplicates

### Bug Description

Sharing a post from the Bluesky network results in empty posts, if the post's permissions are not adjusted.

My default permission is set to public, but this is not recognised when sharing posts from Bluesky. 

### Steps to Reproduce

1.  Set default permission to "Public"
2.  Quote share post from Bluesky

Actual Result: Empty post

Expected Result: Shared post that honours default settings

### Screenshots

![signal-2023-10-08-202252](https://github.com/friendica/friendica/assets/28751375/e7aeea11-5fba-4323-baf5-b01ad50d958c)

### Platform Info

Friendica Version: Current RC

**Comment by annando** [comment]
I don't think that it is some permission thing. It seems that the issue is that you cannot quote share posts from Bluesky, am I right?

**Comment by annando** [comment]
Quote sharing of connector networks always is converted to sharing the external link to that post. This is done intentionally. I guess that this is the case here.
